examples/sphere/shallow_water.jl

```diff
@@ -77,6 +91,17 @@ elseif test_name == steady_state_compact_test_name
     const ϕᵦ = -30.0
     const ϕₑ = 90.0
     const xₑ = 0.3
+elseif test_name == barotropic_instability_test_name
+    const u_max = 80.0
+    const xₑ = 0.3
+    const αₚ = 19.09859
+    const βₚ = 3.81971
+    const h0 = 10158.18617 # value for initial height from Tempest https://github.com/paullric/tempestmodel/blob/master/test/shallowwater_sphere/BarotropicInstabilityTest.cpp#L86
+    const h_hat = 120.0
+    const ϕ₀ = 25.71428
+    const ϕ₁ = 64.28571
+    const ϕ₂ = 45.0
+    const eₙ = exp(-4.0 / (ϕ₁ - ϕ₀)^2)
```

Comment:
Ah I didn't notice this yesterday. I think you need to change `eₙ` or `u_max` as well, as `ϕ₁` and `ϕ₀` are radians in the paper.

Reply:
I'm not sure I follow what you're saying. I did change `ϕ₀` and `ϕ₁` to degrees. And `eₙ` is computed from them. Are you saying that we should change the formula for `eₙ`? 
In the previous case `steady_state_compact`, we didn't change `u0`, which is the analog of `u_max` here and we didn't change the formulation for `uλprime` that has a similar factor `exp(4.0 / xₑ)`. Are you saying we should have change that as well?

Reply:
It's ok in the `steady_state_compact` because `x(ϕprime) = xₑ * (ϕprime - ϕᵦ) / (ϕₑ - ϕᵦ)`, and `u` is a function of x, so the results would be the same as long as the latitudes are all in degrees or all in radians. But here `eₙ = exp(-4.0 / (ϕ₁ - ϕ₀)^2)`, and `u` is affected by `eₙ`, so the results would be different for degrees and radians.